operators/kepler-operator/0.9.0/manifests/kepler-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,338 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [
+        {
+          "apiVersion": "kepler.system.sustainable.computing.io/v1alpha1",
+          "kind": "Kepler",
+          "metadata": {
+            "labels": {
+              "app.kubernetes.io/instance": "kepler",
+              "app.kubernetes.io/name": "kepler",
+              "app.kubernetes.io/part-of": "kepler-operator"
+            },
+            "name": "kepler"
+          },
+          "spec": {
+            "exporter": {
+              "deployment": {
+                "port": 9103
+              }
+            }
+          }
+        }
+      ]
+    capabilities: Basic Install
+    categories: Monitoring
+    containerImage: quay.io/sustainable_computing_io/kepler-operator:0.9.0
+    createdAt: "2023-10-13T05:19:02Z"
+    description: 'Deploys and Manages Kepler on Kubernetes '
+    operators.operatorframework.io/builder: operator-sdk-v1.27.0
+    operators.operatorframework.io/project_layout: go.kubebuilder.io/v3
+    repository: https://github.com/sustainable-computing-io/kepler-operator
+  name: kepler-operator.v0.9.0
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions:
+    owned:
+    - description: Kepler is the Schema for the keplers API
+      displayName: Kepler
+      kind: Kepler
+      name: keplers.kepler.system.sustainable.computing.io
+      statusDescriptors:
+      - displayName: Conditions
+        path: conditions
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:conditions
+      version: v1alpha1
+  description: |-
+    ### Introduction
+    Kepler (Kubernetes Efficient Power Level Exporter) uses eBPF to probe
+    energy-related system stats and exports them as Prometheus metrics.
+
+    ### API Breakage
+    **NOTE:** `0.8.z` is **NOT** backward compatible with `0.7.0` or lower. Before
+    installing `0.8.z`, you must uninstall `0.7.0` or (earlier) completely
+    (including CRDs) by following the steps in the
+    [uninstall operator section](https://github.com/sustainable-computing-io/kepler-operator#uninstall-the-operator)
+
```

Comment:
@stleerh Thanks for raising this (good) point. API Breakage applies to all `minor` version change.  
I think we should make this text generic that the operator does not guarantee API compatibility between `minor` releases - it may or may not work.